### PR TITLE
trivial style fixes

### PR DESCRIPTION
**Pull request by gerkey**
@jacquelinekay , for your consideration

### CODE REVIEW

File: test_rclcpp/test/test_multithreaded.cpp [hunk 1]
```diff
@@ -57,7 +58,8 @@ static inline void multi_consumer_pub_sub_test(bool intra_process)
   rclcpp::executors::MultiThreadedExecutor executor;
   // Try to saturate the MultithreadedExecutor's thread pool with subscriptions
   for (uint32_t i = 0; i < executor.get_number_of_threads(); i++) {
-    auto sub = node->create_subscription<test_rclcpp::msg::UInt32>(node_topic_name, 16, callback, callback_group);
+    auto sub = node->create_subscription<test_rclcpp::msg::UInt32>(node_topic_name, 16, callback,
+        callback_group);
```
Comment (dirk-thomas) [inline test_rclcpp/test/test_multithreaded.cpp]: Please wrap after the open parenthesis instead of in the middle of the args. Same below.

Comment (dirk-thomas) [inline test_rclcpp/test/test_multithreaded.cpp]: Nvm applying the change myself.

File: test_rclcpp/test/test_multithreaded.cpp [hunk 2]
```diff
@@ -77,7 +79,7 @@ static inline void multi_consumer_pub_sub_test(bool intra_process)
   // test spin_some
   // Expectation: The message was published and all subscriptions fired the callback.
   // Use spin_once to block until published message triggers an event
-  //executor.spin_once();
+  // executor.spin_once();
```
Comment (dirk-thomas) [inline test_rclcpp/test/test_multithreaded.cpp]: Why should the code contain all these commented statements in the first place?

Comment (jacquelinekay) [inline test_rclcpp/test/test_multithreaded.cpp]: This is in the category of stuff I will clean up when I consider this test ready for review, which will be when the fixes for multithreading are ready for review.

**Comment by gerkey** [comment]
Feel free to ignore these changes; I knew that you weren't anywhere near done with this test.  I was just making the changes to acquaint myself with our style guide and to exercise the impressive linting test suite.

It led to a good style discussion with @dirk-thomas, and some subsequent fixes and extensions to the [style guide](https://github.com/ros2/ros2/wiki/Developer-Guide#c-1).

**Comment by gerkey** [comment]
Fixed in #72.

**Pull request closed without merging**